lib/getters/shopperContactGetters.ts

```diff
@@ -31,12 +31,12 @@ const getAddressDetails = (shopper: CustomerContact) => {
   return {
     id: getId(shopper) || "",
     firstName: getFirstName(shopper) || "",
-    lastNameOrSurname: getLastName(shopper) || "",
+    lastName: getLastName(shopper) || "",
     address1: getAddressLine1(shopper) || "",
     address2: getAddressLine2(shopper) || "",
-    cityOrTown: getCityOrTown(shopper) || "",
-    stateOrProvince: getStateOrProvince(shopper) || "",
```

Comment:
I think the field name changes in this getter need to be reverted.

Reply:
Done